scripts/globals/mobskills/aqua_blast.lua

```diff
@@ -0,0 +1,33 @@
+-----------------------------------
+--  Aqua Blast
+--
+--  Description: Fires a blast of Water, dealing damage in a fan-shaped area. Additional effect: knockback
+--  Type: Magical (Water)
+--  Utsusemi/Blink absorb: Wipes shadows
+--  Range: Fan (cone)
+--  Note: There was not a lot of information about this spell available online, so
+--        the initial implementation is relatively basic. It was implemented based
+--        heavily off of `aqua_breath.lua`.
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/monstertpmoves")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    -- Do not use this weapon skill on targets behind. Sub-Zero Smash
+    -- should trigger in this case.
+    if target:isBehind(mob) then
+        return 1
+    end
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    local dmgmod = MobBreathMove(mob, target, 0.25, 1.5, xi.magic.ele.WATER, 400)
+    local dmg = MobFinalAdjustments(dmgmod, mob, skill, target, xi.attackType.BREATH, xi.damageType.WATER, MOBPARAM_WIPE_SHADOWS)
```

Comment:
xi.attackType.BREATH <-- Is it?

Reply:
Wow nice find.

Copying for reference from the translation of that link:

"The name means "water gust", and it looks like abreath- type attackthat spits out a stream of water from the mouth , but it seems to be a kind of magic damage because it can be defended inthe sanctuaryof the goddess ."

It also mentions "it has a knockbackeffect over a fairly long distance", so I will update the placeholder to 2 until we can get a packet catpure.

Reply:
I left a link to the mob family in the open issue.
There you can "easily" get some info on its mob skills

Reply:
Very helpful. Thank you!

Reply:
Applied those changes.

One open question: how to determine proper damage values for spells like these? It seems most of the spells using `MobMagicalMove` multiply the `mob:getWeaponDmg()` by some fixed constant (between 2-8) and leave `dmgmod = 1`.

But it's unclear if these are all just arbitrarily chosen, or come from some more definitive source.

I went with something that seemed reasonable given other spell(s) that are similar. But right now it's largely just a "magic number".

Reply:
ok so... taking the words of certain wise man...
as you have probably imagined by now, damage calculations are NOT being handled as they should.
some skills have "random" values to try to come close to what retail does with our calculations.
others use the values from wiki and leave it as is.

Reply:
My process wasn't very scientific.

I tried to pick a reasonable median value from a bunch of similar Water magic spells that use the same constant for the damage. E.g. `regurgitation`, `acid_spray`, `water_ii`, `gastric_bomb`, `aqua_ball`, and more.